docs/admin/reference/hardware.rst

```diff
@@ -52,118 +60,56 @@ The 2nd-generation ThinkPad T14 **with an 11th-generation Intel Core processor**
 
 The Ethernet and Wi-Fi controllers may not work without one-time manual configuration, as documented in the following sections.
 
 Ethernet controller
 ^^^^^^^^^^^^^^^^^^^
-After Qubes starts for the first time, when ``sys-net`` fails to start, follow the instructions below for the :ref:`thinkpad_t490`, but only for the ``dom0:00_1f.6`` Ethernet device.
-
-.. _thinkpad_t490:
-
-Lenovo ThinkPad T490 (with 8th-generation Intel Core processor)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The ThinkPad T490 **with an 8th-generation Intel Core processor** is a recommended option for the SecureDrop Workstation. If you plan to use it, you should follow the instructions below to ensure that the BIOS is up to date and adequately configured before proceeding with the installation.
-
-
-.. caution::
-
-  The versions of the T490 with 10th generation Intel Core processors are at present **untested and unsupported**. The Workstation has been tested on models 20N2002AUS & 20N20046US.
-
-Lenovo ThinkPad T480
-~~~~~~~~~~~~~~~~~~~~
-The ThinkPad T480 is also a recommended option for SecureDrop Workstation, as it is being used by the core team for development and testing. If you plan to use it, you should follow the instructions below to ensure that the BIOS is up to date and adequately configured before proceeding with the installation:
-
-.. _thinkpad_bios:
-
-Upgrading the BIOS on Lenovo ThinkPad laptops
----------------------------------------------
-
-The instructions below assume the use of a Linux-based computer for the creation of a BIOS upgrade USB. To upgrade the BIOS:
-
-- Locate the ThinkPad's "machine type" in its BIOS setup program:
-
-  #. Boot (or reboot) the ThinkPad and follow the prompts to enter setup, usually via the <Enter> and <F1> keys.
-  #. On the **Main** tab, look for the **Machine Type Model**.  The first four characters, such as `20L5`, `20L6`, or `20S0`, are the machine type.
-
-- Visit `<https://support.lenovo.com>`_ in the Linux-based computer. Type the machine type found above into the search bar, then press **Enter**.
-- In the "Product Home" page, select **Drivers And Software** and choose **BIOS/UEFI**.
-- Download the file called either **BIOS Update (Bootable CD)** or **BIOS Update (Utility & Bootable CD)**.
+After Qubes starts for the first time, when ``sys-net`` fails to start, follow the troubleshooting instructions for :ref:`reset_pci`, but only for the ``dom0:00_1f.6`` Ethernet device.
 
-.. note::
-  A Tails USB can be used for the verification and conversion process described below, but the Lenovo support site blocks requests over Tor, preventing the ISO download. To work around this, either:
+.. _framework_13_series:
 
-  - download the BIOS ISO on a different computer and transfer it to Tails using a USB stick, or
-  - download the ISO in Tails using the Unsafe Browser as follows:
+Framework 13 (13th-generation)
```

Comment:
Since it's the only recommendation that doesn't require hardware-specific fixes, maybe we should make it the first one in the list?

Reply:
That makes sense to me - I've pushed an updated commit with this change in place